int16 x_min
int16 y_min
int16 x_max
int16 y_max
float32 confidence
string class_name